pdns/dnspacket.cc

```diff
@@ -81,9 +81,9 @@ const string& DNSPacket::getString()
   return d_rawpacket;
 }
 
-string DNSPacket::getRemote() const
+ComboAddress DNSPacket::getRemote() const
```

Comment:
Won't & give us risk of leaking if we're not careful? CA is pretty small in any case.